atm.js

```diff
@@ -1,21 +1,77 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+//Global Vars
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+var currentCheckBalance = 0;
+var currentSaveBalance = 0;
 
-}
+// event listeners
 
-function checkingWithdrawal(amount){
+// checking deposit button
 
-}
+$('#checkDepositButton').click('click', function(){
```

Comment:
use `.on("click")` instead of `.click("click")`. I wonder what other arguments you can pass to `.click?`